CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 ## vNEXT
 ### Highlights :tada:
++ Add support for bracket syntax to facades in Scala 3. Also, handle `@py.native` annotation in `native_impl` and `native_named_impl` methods
```

Comment:
Rather than having a full item for this, let's just drop the `@PyBracketAccess` caveat from the original Scala 3 item and link to both this and the main PR in that item